src/lang/sclang.js

```diff
@@ -24,6 +25,9 @@ import { SclangResultType } from '../Types';
 // 'any' just opts out of type checking
 type ChildProcessType = any; // child_process$ChildProcess;
 
+const RESULT_LISTEN_HOST = 'localhost';
+const RESULT_LISTEN_PORT = 22967;
```

Comment:
TCP port is hardcoded, and so there could only be one `sclang` running at a time.